python/cudf/cudf/_lib/column.pyx

```diff
@@ -554,10 +554,7 @@ cdef class Column:
                     owner=data_owner,
                     exposed=True,
                 )
-                if isinstance(data_owner, CopyOnWriteBuffer):
-                    data_owner.get_ptr(mode="write")
```

Comment:
Why do we no longer have to mark the `data_owner` as exposed in the `TenableBuffer` case?